preview/galleon-content/src/main/resources/feature_groups/standalone-ha.xml

```diff
@@ -21,7 +21,6 @@
     <feature-group name="management-interfaces"/>
     <feature-group name="access-control"/>
     <feature-group name="health"/>
-    <feature-group name="micrometer" />
```

Comment:
This should be replaced with "metrics", otherwise the non-microprofile standalone configurations will lack something to get metrics in the preview distribution.